lib/Controller/RoomController.php

```diff
@@ -683,7 +692,9 @@ protected function createEmptyRoom(string $roomName, bool $public = true, string
 
 		// Create the room
 		try {
-			$room = $this->roomService->createConversation($roomType, $roomName, $currentUser, $objectType, $objectId);
+			$room = $this->roomService->createConversation($roomType, $roomName, $currentUser, $objectType, $objectId, $password);
+		} catch (PasswordException $e) {
+			return new DataResponse(['reason' => $e->getReason(), 'hint' => $e->getHint()], Http::STATUS_FORBIDDEN);
```

Comment:
I wouldn't go with a new status code
```suggestion
			return new DataResponse(['error' => 'password', 'hint' => $e->getHint()], Http::STATUS_BAD_REQUEST);
```

or is the reason to be displayed additionally? I think it was the english string of the hint.